### PR TITLE
Bug/31427 Incorrect offset for check window dates upon submission

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
Changed presentation to display start and end dates in utc format

### CODE REVIEW

File: admin/services/check-window-v2.service.js [hunk 1]
```diff
@@ -89,43 +89,45 @@ checkWindowV2Service.prepareSubmissionData = (requestData, checkWindowId = null)
   if (checkWindowId) {
     checkWindowData.id = checkWindowId
   }
+  const utcOffsetInHours = moment().utcOffset() / 60
```
Comment (GuyHarwood) [inline admin/services/check-window-v2.service.js]: what unit is `moment().utcOffset()` ?

Comment (GeorgeChatzigiannis) [reply admin/services/check-window-v2.service.js]: @GuyHarwood minutes

Comment (GeorgeChatzigiannis) [reply admin/services/check-window-v2.service.js]: https://momentjs.com/docs/#/manipulating/utc-offset/

**Comment by GeorgeChatzigiannis** [comment]
Closing as this got resolved for now by #1123

**Pull request closed without merging**